source/_integrations/nest.markdown

```diff
@@ -322,9 +322,40 @@ This feature is enabled by the following permissions:
 
 - *Allow Home Assistant to know when there's a camera event*
 - *Allow Home Assistant to know when there's a doorbell event*
-- *Other permissions in the Nest or Google Home apps*.
+- *Other permissions and notification settings in the Nest or Google Home apps*.
 </div>
 
+### Google Home App Notification Settings
+
+The Google Home App Notifications settings control not only which notifications are sent to your phone,
+but also what gets published to the Pub/Sub feed.
+
+For example, if you enable *Away-only notifications*, Home Assistant will only receive events when your phone is away from home.
+
+Another thing that may not be intuitive, is that seeing the event your device history does not mean it's was published to the feed.
+However, if you are getting push notifications the settings are likely working.
+
+Note: The exact settings and affect to feed  may vary by camera model or app version.
+
+
+If you are still not getting notifications you can read this [troubleshooting guide from Google]<!-- textlint-disable -->
+(https://support.google.com/googlenest/answer/9230439#zippy=%2Cyour-camera-detected-something-but-you-didnt-get-a-camera-alert)
+<!-- textlint-enable -->
+
+{% details "Google Home App Notification Settings" %}
+
+
+| Google Home App Setting  | Notes                                                                                 |
+| ------------------------ | :-----------------------------------------------------------------------------------: |
+| Notifications: Push      | Required for detection event to be published                                          |
+| Notifications: Away-Only | Events will only be published when owner is detected as away from home                |
+| Seen: Motion             | Only `motion` events will be published                                                |
```

Comment:
Is it "Only" that those events will be published? Seems like this option isn't exclusive with the others

Reply:
Oops that's not correct.
Revsion here https://github.com/home-assistant/home-assistant.io/pull/29576/commits/f7060e1dab7ebadcd07c4cc066c74a4e0356b8a4